wagtailio/blog/models.py

```diff
@@ -55,9 +73,23 @@ def serve(self, request):
             {
                 "page": self,
                 "posts": posts,
+                "featured_posts": [post.page for post in self.featured_posts.all()],
+                "categories": Category.objects.all()
+                .values_list("pk", "title")
+                .distinct()
```

Comment:
note: distinct is not needed when you query all objects.
suggestion: it would probably be nicer if we returned only categories in use on by `BlogPage`

Reply:
Thanks, updated in cf9286b